firestore/tests/unit/test__helpers.py

```diff
@@ -456,13 +456,16 @@ def test_add_value_at_field_path_failure_adding_more_generic_path(self):
             helper.unpacked_field_paths, {'x': {'y': {'z': helper.PATH_END}}})
 
     def test_parse(self):
+        from google.cloud.firestore_v1beta1._helpers import FieldPath
         # "Cheat" and use OrderedDict-s so that iteritems() is deterministic.
         field_updates = collections.OrderedDict((
             ('a.b.c', 10),
             ('d', None),
             ('e.f1', [u'no', b'yes']),
             ('e.f2', 4.5),
+            (FieldPath.from_string('e.f3').to_api_repr(), (3, 1)),
```

Comment:
Can you add a test case that verifies field path escaping? `FieldPath('h.i')` should do the trick.